tests/components/shelly/test_sensor.py

```diff
@@ -350,9 +350,11 @@ async def test_block_sensor_without_value(
     [
         ("test_name_battery", "98", DEVICE_BLOCK_ID, "battery", None),
         ("test_name_operation", "normal", SENSOR_BLOCK_ID, "sensorOp", "unknown"),
+        ("test_name_operation", "normal", SENSOR_BLOCK_ID, "sensorOp", "normal"),
+        ("test_name_self_test", "pending", SENSOR_BLOCK_ID, "selfTest", "completed"),
```

Comment:
I am a little bit worried if the test is actually working, because it looks like an input of `pending` results in `completed`

Reply:
`pending` is the initial state, `completed` is after:

```python
monkeypatch.setattr(mock_block_device.blocks[block_id], attribute, value)
```